### PR TITLE
[18.0][MIG] document_page_quality_manual: Migration to 18.0

### PR DESCRIPTION
**Pull request by Kimkhoi3010**
Depend on:

- [x] https://github.com/OCA/knowledge/pull/504
- [x] #606

### CODE REVIEW

File: document_page_quality_manual/test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-document_page @ git+https://github.com/OCA/management-system.git@refs/pull/504/head#subdirectory=document_page
```
Comment (xaviedoanhduy) [inline document_page_quality_manual/test-requirements.txt]:
```suggestion
odoo-addon-document_page @ git+https://github.com/OCA/knowledge.git@refs/pull/504/head#subdirectory=document_page
```

**Review by xaviedoanhduy — CHANGES_REQUESTED** [review]

File: document_page_quality_manual/readme/CONFIGURE.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+To enter your work instruction template, you need to:
+
+- go to Management Systems \> Documentation \> Work Instructions
```
Comment (ajaniszewska-dev) [inline document_page_quality_manual/readme/CONFIGURE.md:3]: i find it misleading/too general, as per usage.md we need to tick this option image

File: document_page_quality_manual/readme/CONFIGURE.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+- select the Work Instructions category and edit the template
```
Comment (max3903) [inline document_page_quality_manual/readme/CONFIGURE.md]:
```suggestion
To edit the quality manual template:

- Go to Management Systems \> Configuration \> Categories
- Select the Quality Manual category
- Edit the content
```

File: document_page_quality_manual/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Quality Manual",
+    "version": "18.0.1.0.0",
+    "category": "Management System",
+    "author": "OpenERP SA, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/management-system",
+    "license": "AGPL-3",
+    "depends": ["document_page"],
```
Comment (max3903) [inline document_page_quality_manual/__manifest__.py]:
```suggestion
    "depends": ["mgmtsystem_manual"],
```

File: document_page_quality_manual/readme/CREDITS.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
+
```
Comment (max3903) [inline document_page_quality_manual/readme/CREDITS.md]:
```suggestion
```

File: document_page_quality_manual/readme/USAGE.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+To enter your quality manual, you need to:
+
+- go to Knowledge \> Pages \> Pages
+- Create a new page and select the Quality Manual category.
```
Comment (max3903) [inline document_page_quality_manual/readme/USAGE.md]:
```suggestion
To enter your quality manual:

- Go to Management Systems \> Documentation \> Manuals
- Create a new page
- Enter a title
- Select the Quality Manual category
- Fill in the content
```

File: document_page_quality_manual/static/description/icon.png
Comment (max3903) [inline document_page_quality_manual/static/description/icon.png:1]: @Kimkhoi3010 Please update the icon using this one:
https://github.com/OCA/management-system/blob/17.0/mgmtsystem/static/description/icon.png

File: document_page_quality_manual/images/wiki_pages_quality_manual.jpeg
Comment (ajaniszewska-dev) [inline document_page_quality_manual/images/wiki_pages_quality_manual.jpeg:1]: could we update this with some new screenshot?

File: document_page_quality_manual/static/description/icon.png
Comment (Kimkhoi3010) [reply document_page_quality_manual/static/description/icon.png:1]: thank for your time !

File: document_page_quality_manual/images/wiki_pages_quality_manual.jpeg
Comment (Kimkhoi3010) [reply document_page_quality_manual/images/wiki_pages_quality_manual.jpeg:1]: thank for your time !

File: document_page_quality_manual/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    "version": "18.0.1.0.0",
+    "category": "Management System",
+    "author": "OpenERP SA, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/management-system",
+    "license": "AGPL-3",
+    "depends": ["mgmtsystem_manual"],
+    "data": ["data/document_page.xml"],
+    "installable": True,
+    "auto_install": False,
+    "images": ["images/wiki_pages_quality_manual.jpeg"],
```
Comment (max3903) [inline document_page_quality_manual/__manifest__.py]:
```suggestion
    "images": ["static/description/quality_manual_form.png"],
```

File: document_page_quality_manual/images/wiki_pages_quality_manual.jpeg
Comment (max3903) [inline document_page_quality_manual/images/wiki_pages_quality_manual.jpeg:1]: @Kimkhoi3010 Please delete this file.

**Review by max3903 — CHANGES_REQUESTED** [review]
Still need some fixes and squashing migration commits

**Review by Kimkhoi3010 — COMMENTED** [review]
@max3903 Why you delete test-requirements.txt when we need it to pass test case?

**Comment by max3903** [comment]
@Kimkhoi3010 Because we are ready to merge.

**Pull request merged**